src/riak_cs_block_server.erl

```diff
@@ -51,7 +51,8 @@
          format_status/2]).
 
 -record(state, {riak_client :: riak_client(),
-                close_riak_connection=true :: boolean()}).
+                close_riak_connection=true :: boolean(),
+                manifest :: lfs_manifest()}).
```

Comment:
I would prefer just putting `BagId` here, to keep block server as clean as we can. This will also prevent unnecessary confusion in future and utilzing other members in `lfs_manifest_v3{}`.